app/services/concerns/xccdf/util.rb

```diff
@@ -17,13 +17,15 @@ module Util
       include ::Xccdf::RuleResults
 
       def save_all_benchmark_info
-        save_benchmark
-        save_profiles
-        save_rules
-        save_rule_identifiers
-        save_profile_rules
-        save_rule_references
-        save_rule_references_rules
+        unless benchmark_saved? # rubocop:disable Style/GuardClause
```

Comment:
Rubocop is confused here. idk why it's trying to make me move this to be a guard clause when the block is multiple lines long :man_shrugging:

Reply:
I don't know what does it have to do with the length of the block? It just wants to return early and have the rest of the method unindented I think  
```
return unless benchmark_saved?
```

Reply:
Ohh, yeah that makes sense :)